services/QuillLMS/client/app/bundles/Shared/components/feedback/conceptExplanation.tsx

```diff
@@ -7,16 +7,22 @@ function getClassName(description, leftBox, rightBox) {
   return "concept-explanation empty"
 }
 
-const ConceptExplanation = ({ description, leftBox, rightBox, translatedDescription, translated}) => (
-  <div className={getClassName(description, leftBox, rightBox)}>
-    <div className="concept-explanation-title"><img alt="Light Bulb Icon" src="https://assets.quill.org/images/icons/hint.svg" /><span>Hint</span></div>
-    <div className="concept-explanation-description" dangerouslySetInnerHTML={{__html: description}} />
-    { translated && translatedDescription && <div className="concept-explanation-translation" dangerouslySetInnerHTML={{__html: translatedDescription}} /> }
-    <div className="concept-explanation-see-write">
-      <div className="concept-explanation-see" dangerouslySetInnerHTML={{__html: leftBox}} />
-      <div className="concept-explanation-write" dangerouslySetInnerHTML={{__html: rightBox}} />
+const ConceptExplanation = ({ description, leftBox, rightBox, translatedDescription, translated}) => {
+  const urlParams = new URLSearchParams(window.location.search)
+  const translatedOverride = urlParams.get('showTranslations')
+  const showsTranslation = translatedDescription && (translated || translatedOverride)
```

Comment:
One thing to note is that I think `translatedOverride` here is going to evaluate to `true` even if the querystring is `?showTranslations=false`, because the value is always a string. That probably doesn't matter in practice but it may be worth explicitly checking that it is equal to true?

Reply:
Also, a more general question here: do we always want the translation to show if the query string is present, even if `translated` is false? Maybe I'm misinterpreting something but it seems to me like maybe we want the query string to create the condition where it is possible to show the translations, but we should still respect the user's toggle?

Reply:
The `translated` flag only comes in on the admin interface. So this is a different override to show translations for students. The `translatedDescription` is the thing that figures out if there is an actual translation to show. Your comment indicates to me that these things need better naming!